### PR TITLE
DDCE-1386: Implement new error page for switched-over users

### PR DESCRIPTION
**Pull request by abhijeetardale**
# DDCE-(StoryNumberHere)

##Bug fix / New feature (delete as appropriate)

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## PR Suggestions
- Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?
- Have you done a visual check of the changes?
- Is there any still commented or unused code? Lingering printlns?
- Have things been implemented in a complicated way?
- Are the test still over the coverage threshold?
- Does the compiler throw a lot of warnings? 
- Have methods and tests been named clearly?
- Were there any changes in the config? Do changes need to be made in app-config-???

## Checklist PR Raiser
 - [ ]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [ ]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

## Checklist PR Reviewer
 - [ ]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [ ]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: app/controllers/SwitchOverErrorController.scala [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import views.html.SwitchOverErrorView
+
+import javax.inject.Inject
+import scala.concurrent.Future
+
+
+class SwitchOverErrorController @Inject()(
+    identify: AuthIdentifierAction,
+    getData: UserDataRetrievalAction,
```
Comment (RafSobol) [inline app/controllers/SwitchOverErrorController.scala:30]: now I think about it, do we even need a the UserDataRetrievalAction in there? happy to keep it, but there's a few places where we don't need it I think

File: conf/messages.cy [hunk 2]
```diff
@@ -269,6 +269,14 @@ cannotFindApplication.heading = (W)We cannot find your application(W)
 cannotFindApplication.p1 = (W)We save unfinished applications for 28 days. After that, you need to(W)
 cannotFindApplication.p1.link = (W)start your application again(W)
 
+# Switch over error page Messages
+# ----------------------------------------------------------
+switchOverError.title = (W)Sorry, there is a problem with some of your answers - Error(W)
+switchOverError.heading = (W)Sorry, there is a problem with some of your answers(W)
+switchOverError.p1 = (W)We have redesigned this service since you last worked on your application.(W)
+switchOverError.p2 = (W)We have brought some of your answers across to the new service. However some of your answers have been lost.(W)
```
Comment (RafSobol) [inline conf/messages.cy:277]: I believe this 'However' should be followed by a comma - asked Mark, he'll check with a content designer

File: app/controllers/SwitchOverErrorController.scala (hunk 1 above)
Comment (abhijeetardale) [reply app/controllers/SwitchOverErrorController.scala]: updated now

File: conf/messages.cy (hunk 2 above)
Comment (abhijeetardale) [reply conf/messages.cy:277]: will be updated once confimred

**Review by RafSobol — APPROVED** [review]

**Review by RafSobol — APPROVED** [review]

**Pull request merged**